rpcadaptors.go

```diff
@@ -355,6 +355,14 @@ func (b *rpcSyncMgr) ProcessTransaction(tx *dcrutil.Tx, allowOrphans bool,
 		rateLimit, allowHighFees, tag)
 }
 
+// RecentlyConfirmedTxn returns with high degree of confidence whether a
+// transaction has been recently confirmed in a block.
+//
+// This method may report a false positive, but never a false negative.
+func (b *rpcSyncMgr) RecentlyConfirmedTxn(hash *chainhash.Hash) bool {
+	return b.syncMgr.RecentlyConfirmedTxn(hash)
```

Comment:
This can just be `return b.server.recentlyConfirmedTxns.Contains(hash[:])`.  Then there is no need to export a new method from the internal sync manager as mentioned in the other comment.